app/utils.py

```diff
@@ -74,6 +75,14 @@ def sanitize_email(email_address: str, not_lower=False) -> str:
     return email_address
 
 
+def sanitize_next_url(url: Optional[str]) -> Optional[str]:
```

Comment:
I think we also want to handle the case next_url is an absolute url, for example https://app.simplelogin.io/something

Reply:
Added support for controlled absolute urls :+1: